platform/default/src/mbgl/layermanager/layer_manager.cpp

```diff
@@ -77,8 +77,6 @@ void LayerManagerDefault::addLayerType(std::unique_ptr<LayerFactory> factory) {
     std::string type{factory->getTypeInfo()->type};
     if (!type.empty()) {
         typeToFactory.emplace(std::make_pair(std::move(type), factory.get()));
-    } else {
```

Comment:
I guess this is a valid requirement for a layer type to provide a name, why this logging is to be removed?

Reply:
according to @alexshalamov custom layer atm needs to have no name

Reply:
We should either remove log or add name for a custom layer's type info. I'm fine with both solutions.

Reply:
let's add a name

Reply:
added "custom"